README.md

```diff
@@ -1,209 +1,135 @@
-Code will be build and tests run everytime you save a file:
+Reads the configuration and starts relaying packets.
 
-```bash
-# in one console
-yarn watch:build
+- run `ibc-relayer --help` to print usage
+- [ibc-relayer spec](spec/ibc-relayer.md)
 
-# in another console
-yarn watch:test
-```
+## Quick start
 
-### Test Framework
+### Configure and start the relayer
 
-I am using [Ava](https://github.com/avajs/ava) here simply because that was automatically set up with
-the TypeScript helper. Rather than immediately rush to the well-known jasmine from CosmJS, I tried it out
-and so far like the functionality. Nice test watcher, clean error messages, and clickable links right to
-the failing source. Let's try this out a bit on this project.
+1. Init the configuration
 
-You will want to read [how to write tests](https://github.com/avajs/ava/blob/master/docs/01-writing-tests.md)
-and maybe a little [the list of valid assertions](https://github.com/avajs/ava/blob/master/docs/03-assertions.md#built-in-assertions)
-before coding.
+   ```sh
+   ibc-setup init --src relayer_test_1 --dest relayer_test_2
+   ```
 
-### Protobuf
+   - creates relayer's home directory at `~/.ibc-setup`
+   - creates `app.yaml` inside relayer's home with `src`, `dest` and newly generated `mnemonic`
+     - [What is app.yaml?]()
+   - pulls default `registry.yaml` to relayer's home
+     - [What is registry.yaml?]()
+   - funds addresses on both sides so relayer can pay the fee while relaying packets
 
-To re-build the protobuf definitions, first look at [scripts/proto/env](./scripts/proto/env) and ensure you
-have set the desired Cosmos SDK tag there (eg. `v0.41.0`). After that, run:
+   > **NOTE:** Run `ibc-setup balances` to see the amount of tokens on each address.
 
-```bash
-./scripts/proto/get-proto.sh
-./scripts/proto/define-proto.sh
-```
+2. Create `ics20` channel
 
-This will overwrite the data in `src/codec` with newly generated definitions. We delete the folder first
-to avoid outdated artifacts, meaning any manual changes will be lost.
+   ```sh
+   ibc-setup ics20
+   ```
 
-### Changelog and Publishing
+   - creates a new connection on source and desination chains
+   - saves connection ids to `app.yaml` file
+   - creates a new channel
 
-Please use [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) for all commit messages. Some basic examples:
-
-```
-feat(api): add client upgrade to API
-docs: update README
-fix(client): properly reconnect
-refactor!: Rewrite public API
-```
+3. Start the relayer in the verbose mode and 10s frequency polling
+   ```sh
+   ibc-relayer start -v --poll 10
+   ```
 
-Examples:
+### Send tokens between chains
 
-```
-# normal flow
-yarn build
-yarn test
-yarn fix
+1. Make sure `wasmd` binary is installed on your system
 
-# advanced checks
-yarn cov
-yarn doc
+   - visit [https://github.com/CosmWasm/wasmd](https://github.com/CosmWasm/wasmd) and follow quick start instructions
```

Comment:
Maybe we could make this part more explicit. I wasn't too sure what is the `quickest` thing to do here. Should I use the dev server specified in the docs and start a chain on `:26657` & `16657` ->  https://github.com/CosmWasm/wasmd#dev-server ?

Reply:
It is good to link this as a reference, but we could just be more explicit here. Maybe add this:

1. Ensure you have Go 1.15 installed on your system and that you are running Linux or OSX on amd64 (not arm64 / Mac M1).
2. Ensure that your PATH is set up to include Go binaries  (normally `~/go/bin`)

```
git clone https://github.com/CosmWasm/wasmd.git
cd wasmd
git checkout v0.15.1
make install
```

Reply:
And we can update to v0.16.0 when that is ready

Reply:
@seantking The `wasmd` is used here only to generate new accounts and send transactions. There is no need to run any chains locally to complete the quick start.

I added a note about this and improved the `wasmd` installation part: 799b1b3

Reply:
https://github.com/confio/ts-relayer/blob/799b1b3faae1d4f0bb2162f5ef2164956feae5fb/README.md

Reply:
Nice, makes sense. :+1:  I think having a local quickstart somewhere would be beneficial also.